### PR TITLE
Fix: Include additional Files flag

### PR DESCRIPTION
**Pull request by FrankEssenberger**
## Proposed Changes

We introduced a new flag in the SDK generator. This adjusts the CLI accordingly. So all changes with `additionalFiles` refer to this. I addition I also stabalized the e2e tests. I did the following:
- Made delete of big data async
- Updated to jest 26 (not tested with ts-node officially) but was more stable with respect to teardown
- Used fs.rmDir if node > 12 then it also supports recursive delete. Otherwise you rimraf. 
- Split long running tests tin separate files so that fewer data is deleted created.

## Checklist

- [x] I have added or adjusted tests that prove my fix is effective or that my feature works
- [x] I have added or adjusted documentation

### CODE REVIEW

**Comment by CharlesDuboisSAP** [comment]
npm run test:full
image

**Comment by CharlesDuboisSAP** [comment]
image

**Comment by CharlesDuboisSAP** [comment]
Are these "real problems" that need to be fixed and if yes why doesn't the pipeline pickup the same problems as me?
(I just downloaded a fresh install of your branch in case this is relevant)

**Comment by FrankEssenberger** [comment]
I am happy that the tests in the pipe are green. You are right we should get this also stable on local machines, but it is not reproducible for me. So I would not do something at the moment.

File: test/test-utils.ts [hunk 1]
```diff
@@ -21,3 +21,36 @@ export function getCleanProjectDir(pathPrefix: string, name: string): string {
...
+          reject(`Error in deleting: ${path} with ${err.message}`);
+        } else {
+          resolve(`Deletion of ${path} finished.`);
+        }
+      })
+    );
+  }
+
+  return new Promise<string>((resolve, reject) => {
+    rm(path, { maxBusyTries: busyRetries }, err => {
```
[the remark this answers is not in the document]
Comment (marikaner) [reply test/test-utils.ts:48]: Actually - now that I thought about it a bit more, I get that we can use the node 12 features directly, but I think we should decide for only one of the options. Either 1. we say we only support node >= 12 and use the native remove or 2. we also need to support node 10 and therefore use rimraf.
If there is another reason (e. g. if the async rimraf is also flaky) we should add this to the comment.

**Pull request merged**